packages/devtron-fe-common-lib/tsconfig.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
- clean up tsconfig
- add the tsconfig at the root by extending the one from dashboard
- Check for alias at the root level

Reply:
Let's common out the tsconfig and extend each of the package